packages/web-components/fast-components-msft/package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "@microsoft/fast-components-msft",
+    "description": "A library of Web Components",
```

Comment:
Should we specify these are Microsoft styles?

Reply:
yes